### ISSUE TITLE
[feature request] visualization of wireless node/system range

### ISSUE THREAD

**Issue by corhean** [refinedmods/refinedstorage#2721]
Not an issue - a feature request for a button that would visualize the range of the wireless network and/or an individual wireless transmitter.

**Comment by JRCLARK25** [comment]
Yes please! And/or Visual sphere toggle-able through antenna for laying out extended network with 20+ antennas!

**Comment by raoulvdberge** [comment]
While it sounds interesting, I think it's a bit out of scope for RS. There are other mods that offer utilities for measuring distance etc.
